src/zm_monitor.cpp

```diff
@@ -93,9 +93,13 @@ std::string load_monitor_sql =
 "`ImageBufferCount`, `MaxImageBufferCount`, `WarmupCount`, `PreEventCount`, `PostEventCount`, `StreamReplayBuffer`, `AlarmFrameCount`, "
 "`SectionLength`, `MinSectionLength`, `FrameSkip`, `MotionFrameSkip`, "
 "`FPSReportInterval`, `RefBlendPerc`, `AlarmRefBlendPerc`, `TrackMotion`, `Exif`,"
-"`RTSPServer`, `RTSPStreamName`,"
+"`RTSPServer`, `RTSPStreamName`, `ONVIF_Alarm_Txt`,"
 "`ONVIF_URL`, `ONVIF_Username`, `ONVIF_Password`, `ONVIF_Options`, `ONVIF_Event_Listener`, `use_Amcrest_API`, "
 "`SignalCheckPoints`, `SignalCheckColour`, `Importance`-1, ZoneCount FROM `Monitors`";
+//"`SignalCheckPoints`, `SignalCheckColour`, `Importance`-1, ZoneCount FROM `Monitors`";
```

Comment:
Why are you commenting out this stuff?

Reply:
An old habit of mine, when I change a line of code, I leave the original line commented out... I was going to put the ONVIF-Alarm-Txt at this line, so I commented out that line and changed my mind and forgot to delete the commented line... as you can see is the same... sorry